web/webpack.config.js

```diff
@@ -27,7 +29,7 @@ module.exports = (env) => {
                     test: /\.(js|jsx|ts|tsx)$/,
                     exclude: /node_modules/,
                     use: {
-                        loader: 'ts-loader',
+                        loader: 'babel-loader',
```

Comment:
We're using Babel to transpile our tsx -> cjs in the tests, might as well use the same compile tools we've already setup there for the prod build.